app/src/main/java/com/ayush/weatherapp/main/MainActivity.java

```diff
@@ -9,16 +9,22 @@
 import android.view.Menu;
 import android.view.MenuInflater;
 import android.view.MenuItem;
+import android.widget.TextView;
 import android.widget.Toast;
 import butterknife.BindView;
 import com.ayush.weatherapp.BaseActivity;
 import com.ayush.weatherapp.R;
 
-public class MainActivity extends BaseActivity {
+public class MainActivity extends BaseActivity implements MainContract.View {
 
   @BindView(R.id.layout_drawer) DrawerLayout drawerLayout;
   @BindView(R.id.nav_view) NavigationView navigationView;
   @BindView(R.id.toolbar) Toolbar toolbar;
+  @BindView(R.id.txt_main) TextView textView;
+
+  ActionBar actionbar;
```

Comment:
Could this be done in base activity and the derived activity only provided the title for the action bar.

Reply:
:ok:

Reply:
:ok: